examples/workflows/upstream/template_defaults.py

```diff
@@ -0,0 +1,44 @@
+from hera.workflows import Container, Step, Steps, Workflow
+from hera.workflows.models import IntOrString, RetryStrategy, Template
+
+with Workflow(
+    api_version="argoproj.io/v1alpha1",
+    kind="Workflow",
+    annotations={
+        "workflows.argoproj.io/description": "Template defaults will provide the fixability to configure the defaults values for all templates in workflow.\nIndividual template can be overide default values.\n",
+        "workflows.argoproj.io/version": ">= 3.1.0",
+    },
+    generate_name="template-defaults-",
+    entrypoint="main",
+    template_defaults=Template(
+        retry_strategy=RetryStrategy(
+            limit=IntOrString(
+                __root__="2",
+            ),
+        ),
+        timeout="30s",
+    ),
+) as w:
+    with Steps(
+        name="main",
+    ) as invocator:
```

Comment:
Is this the name we want to go with? I don't remember seeing `invocator` in examples before.